deployment/logger/single_file_logger.go

```diff
@@ -31,7 +31,7 @@ var _ corelogger.Logger = (*SingleFileLogger)(nil)
 func NewSingleFileLogger(tb testing.TB) *SingleFileLogger {
 	// Our logs will go here so GH can upload them:
 	baseDir := "logs"
-	timeLayout := "2006-01-02T15-04-05"
+	timeLayout := "2006-01-02T15:04:05.000"
```

Comment:
Would it make sense to use `time.RFC3339Nano`? https://pkg.go.dev/time#pkg-constants
`RFC3339Nano = "2006-01-02T15:04:05.999999999Z07:00"`